src/config/server.ts

```diff
@@ -71,6 +71,13 @@ const SERVER_CONFIG: StrictServerConfiguration = {
     maxRotatedPerCycle: 1,
     flexibleRotationDelta: 1,
     flexibleRotationEnabled: false,
+    enableProblematicNodeRemoval: true,
```

Comment:
please set this to false for what we merge, then we will either put this in a migration or just set it dynamically after 100% of nodes are 1.16.2

Reply:
sounds good. Ill put the onCycle to 20k too just so when its turned on, it will still have the delayed fuse.